pkg/innerring/processors/container/process_eacl.go

```diff
@@ -99,12 +100,27 @@ func (cp *Processor) approveSetEACL(e container.SetEACL) {
 }
 
 func validateEACL(t *eacl.Table) error {
+	var b big.Int
 	for _, record := range t.Records() {
 		for _, target := range record.Targets() {
 			if target.Role() == eacl.RoleSystem {
 				return errors.New("it is prohibited to modify system access")
 			}
 		}
+		for _, f := range record.Filters() {
+			//nolint:exhaustive
```

Comment:
why not two `if`s or `default`?

Reply:
not two ifs cuz more than 2 values, not default cuz it'd be empty. IMO that's not the case where linter suggests smth meaningful

Reply:
> not two ifs cuz more than 2 values

every `case` has `return`, still can be with `if`s

> not default cuz it'd be empty

more regular expression for a regular go reader (even more if a comment is just `nolint` without some real comment). both are one line long (`deafult` always shorter)

Reply:
well, can be merged, but i do not understand it (not disagree just don't understand why it is easier to do than one of my suggestion)

Reply:
> more regular expression for a regular go reader

empty `default` is useless and always skip, that's what i see as a regular Go reader